client-sdk/ts-web/core/src/consensus.ts

```diff
@@ -78,6 +82,8 @@ export const CODE_INSUFFICIENT_FEE_BALANCE = 2;
  */
 export const CODE_GAS_PRICE_TOO_LOW = 3;
 
+// %%% method priority?
```

Comment:
I don't think method priority needs to be exposed in the client SDK as there is nothing there that the clients can do specifically.

Reply:
yeah agreed